etk-asm/src/ingest.rs

```diff
@@ -141,8 +141,11 @@ struct Root {
 
 impl Root {
     fn new(mut file: PathBuf) -> Result<Self, Error> {
-        // Pop the filename.
-        if !file.pop() {
+        if file.is_file() {
```

Comment:
Will this work for fake files? I'm concerned about [`Ingest::ingest(&self, path, src)`](https://docs.rs/etk-asm/0.1.0/etk_asm/ingest/struct.Ingest.html#method.ingest) specifically.

Reply:
Woops, I should have refreshed before replying ^^;;

Reply:
All tests pass if paired with `file.is_relative()` but realized that this might need a bit more testing.